Task09_Cycle/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+
+
+namespace Task09_Cycle
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Вариант 21. Задание 1");
+            Console.WriteLine("Введите n");
+ 
+            int n = int.Parse(Console.ReadLine());
+            double finalCount = 0;
+
+            for (int i = 1; i <= n; i++)
+                finalCount += 1.0 / i;
+
+            Console.WriteLine(finalCount + "\n");
+
+
+
+
+            Console.WriteLine("Задание 2");
+            Console.WriteLine("Введите n");
+
+            int nSecondTask = int.Parse(Console.ReadLine());
+            int firstFibonacciNumber = 1;
+            int secondFibonacciNumber = 1;
+            int iSecondTask = 3;
+
+            while (iSecondTask <= nSecondTask)
+            {
+                int copyFirstFibonacciNumber = firstFibonacciNumber;
+                firstFibonacciNumber = secondFibonacciNumber;
+                secondFibonacciNumber += copyFirstFibonacciNumber;
+                iSecondTask++;
+            }
+
+            Console.WriteLine(secondFibonacciNumber + "\n");
+
+
+
+
+            Console.WriteLine("Задание 3");
+            Console.WriteLine("Введите число");
+
+            int naturalNumber = int.Parse(Console.ReadLine());
+            int result = 0;
+
+            for(int i=2; i < naturalNumber; i++)
+                if(naturalNumber % i == 0)
+                {
+                    result = i;
+                    break;
+                }
```

Comment:
Небольшая ошибочка: для простых чисел программа выдает неверный результат. Вот листинг
```
Задание 3
Введите число
17
0
```
Разумеется, что должно быть напечатано 17, а не 0.

Можно было попроще и правильнее.
```suggestion
            int result = 0;

            for (result = 2; result <= naturalNumber; result++)
                if (naturalNumber % result == 0)
                    break
```